### PR TITLE
Redirect to sandbox wizard when deleting sandbox

### LINKED ISSUE #1312: Redirect to template selection when deleting sandbox
# 🌈 Feature

Currently, when you delete a sandbox, the application redirects to `/s/new`, which is the template for a React application.

I think that it would be nice to redirect to `/s` (the "template selection" route) instead.

Alternatively, deleting could redirect to the new template page for the type of sandbox that was deleted.

I prefer redirecting to the template selection route. If I delete a sandbox, I don't necessarily want to create a new one right away, so it feels a bit incongruous to be taken to a new sandbox.

### PR DESCRIPTION
**Pull request by pshrmn**
**What kind of change does this PR introduce?**

This PR changes where the user is redirected after deleting a sandbox.

Fixes #1312 

**What is the current behavior?**

The user is redirected to the new React template (`/s/new`).

**What is the new behavior?**

The user is redirected to the sandbox wizard (`/s`).

**Checklist**:

- [ ] Documentation
- [ ] Tests
- [x] Ready to be merged 
- [ ] Added myself to contributors table

### CODE REVIEW

File: packages/app/src/app/store/providers/Router.js [hunk 1]
```diff
@@ -10,6 +10,9 @@ export default Provider({
   redirectToNewSandbox() {
     history.push('/s/new');
   },
+  redirectToSandboxWizard() {
+    history.replace('/s');
```
Comment (lbogdan) [inline packages/app/src/app/store/providers/Router.js]: Hey @pshrmn , can you please make it `/s/`, to prevent the redirect?

Comment (pshrmn) [reply packages/app/src/app/store/providers/Router.js]:
```suggestion
    history.replace('/s/');
```

**Comment by CompuIves** [comment]
Thanks a lot @pshrmn!

**Comment by lbogdan** [comment]
There were still lint errors... 😟

**Comment by CompuIves** [comment]
Fixed!

**Pull request merged**